meinberlin/apps/projects/migrations/0007_create_insights.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from collections import defaultdict
+from django.db import migrations
+
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_insights(apps, schema_editor):
+    ProjectInsight = apps.get_model("meinberlin_projects", "ProjectInsight")
+    Comment = apps.get_model("a4comments", "Comment")
+    Answer = apps.get_model("a4polls", "Answer")
+    Vote = apps.get_model("a4polls", "Vote")
+    Rating = apps.get_model("a4ratings", "Rating")
+    Proposal = apps.get_model("meinberlin_budgeting", "Proposal")
+    Idea = apps.get_model("meinberlin_ideas", "Idea")
+    Like = apps.get_model("meinberlin_likes", "Like")
+    LiveQuestion = apps.get_model("meinberlin_livequestions", "LiveQuestion")
+    MapIdea = apps.get_model("meinberlin_mapideas", "MapIdea")
+
+    insights = defaultdict(ProjectInsight)
+    user_ids = defaultdict(set)
+
+    logger.info("creating insights")
```

Comment:
I'd actually prefer not to have that logging, it doesn't really help, django already tells us what migration is currently running :)